network/basic/chaincode/supplychaincc/ledger-api/constants.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const DonorMSP = 'DonorMSP';
+const ImpactMSP = 'ImpactMSP';
+const HopistalMSP = 'HospitalMSP';
+const ManufacturerMSP = 'ManufacturerMSP';
+const BorderControlMSP = 'BorderControlMSP';
+const StorageFacilityMSP = 'StorageFacilityMSP'
+const MophMSP = 'MOPHMSP'
+
+const ALL_MSPS = [DonorMSP, ImpactMSP, HopistalMSP, ManufacturerMSP, BorderControlMSP, StorageFacilityMSP, MophMSP];
+
+const QUERY_FUNCTIONS = ['query', 'indexOrderDelivery', 'getAllOrderDeliveries'];
+const ALL_FUNCTIONS = [...QUERY_FUNCTIONS, 'issue', 'storageDelivery', 'storageArrival', 'hospitalDelivery', 'hospitalArrival'];
+const BORDERCONTROL_FUNCTIONS = ['storageDelivery'];
+const STORAGEFACILITY_FUNCTIONS = ['storageArrival','hospitalDelivery'];
+const HOSPITAL_FUNCTIONS = [...QUERY_FUNCTIONS, 'hospitalArrrival'];
```

Comment:
Each should be able to get the orders that are on their way to their premises and that are currently in their premises to be able to modify their status